src/main/java/com/celements/mandatory/FileBaseTag1.java

```diff
@@ -100,40 +112,27 @@ boolean isSkipCelementsFileBaseTag1() {
   }
 
   boolean noMainWiki() {
-    String wikiName = getContext().getDatabase();
-    LOGGER.trace("noMainWiki for database [" + wikiName + "].");
-    return (wikiName != null) && !wikiName.equals(getContext().getMainXWiki());
+    return !modelUtils.isMainWiki(context.getWikiRef());
   }
 
   void checkFileBaseTag1() throws XWikiException {
     DocumentReference fileBaseTag1Ref = getFileBaseTag1Ref(getContext().getDatabase());
-    XWikiDocument fileBaseTag1Doc;
-    if (!getContext().getWiki().exists(fileBaseTag1Ref, getContext())) {
-      LOGGER.debug("FileBaseTag1Document is missing that we create it. ["
-          + getContext().getDatabase() + "]");
-      fileBaseTag1Doc = new CreateDocumentCommand().createDocument(fileBaseTag1Ref,
-          _FILE_BASE_TAG_PAGE_TYPE);
-    } else {
-      fileBaseTag1Doc = getContext().getWiki().getDocument(fileBaseTag1Ref, getContext());
-      LOGGER.trace("FileBaseTag1Document already exists. [" + getContext().getDatabase() + "]");
-    }
-    if (fileBaseTag1Doc != null) {
-      boolean dirty = checkPageType(fileBaseTag1Doc);
-      dirty |= checkMenuItem(fileBaseTag1Doc);
-      dirty |= checkMenuName(fileBaseTag1Doc, "en", ".zip");
-      dirty |= checkMenuName(fileBaseTag1Doc, "de", ".zip");
-      if (dirty) {
+    XWikiDocument fileBaseTag1Doc = modelAccess.getOrCreateDocument(fileBaseTag1Ref);
+    boolean dirty = checkPageType(fileBaseTag1Doc);
+    dirty |= checkMenuItem(fileBaseTag1Doc);
+    dirty |= checkMenuName(fileBaseTag1Doc, "en", ".zip");
+    dirty |= checkMenuName(fileBaseTag1Doc, "de", ".zip");
+    if (dirty) {
+      try {
         LOGGER.info("FileBaseTag1Document updated for [" + getContext().getDatabase() + "].");
-        getContext().getWiki().saveDocument(fileBaseTag1Doc, "autocreate"
-            + " Content_attachments.FileBaseTag1.", getContext());
+        modelAccess.saveDocument(fileBaseTag1Doc, "autocreate Content_attachments.FileBaseTag1.");
         treeNodeCache.flushMenuItemCache();
-      } else {
-        LOGGER.debug("FileBaseTag1Document not saved. Everything uptodate. ["
-            + getContext().getDatabase() + "].");
+      } catch (DocumentSaveException dse) {
+        throw new XWikiException(0, 0, "failed saving", dse);
       }
     } else {
-      LOGGER.trace("skip checkFileBaseTag1 because fileBaseTag1Doc is null! ["
-          + getContext().getDatabase() + "]");
+      LOGGER.debug("FileBaseTag1Document not saved. Everything uptodate. ["
+          + getContext().getDatabase() + "].");
```

Comment:
```suggestion
      LOGGER.debug("FileBaseTag1Document not saved. Everything uptodate. [{}]",
      getContext().getDatabase());
```
why not using slf4 message formatting?